src/Bindings/ManualBindings_World.cpp

```diff
@@ -380,9 +378,9 @@ static int tolua_cWorld_QueueTask(lua_State * tolua_S)
 		return cManualBindings::lua_do_error(tolua_S, "Error in function call '#funcname#': Could not get function reference of parameter #1");
 	}
 
-	auto task = std::make_shared<cLuaWorldTask>(*Plugin, FnRef);
-	Plugin->AddResettable(task);
-	self->QueueTask(task);
+	auto ResettableTask = std::make_shared<cLuaWorldTask>(*Plugin, FnRef);
+	Plugin->AddResettable(ResettableTask);
+	self->QueueTask(std::bind(&cLuaWorldTask::Run, ResettableTask, std::placeholders::_1));
```

Comment:
But I think the current state is correct? It's unlikely to change also.

Attempt:
`cpp
class cLuaWorldTask :
	public cPluginLua::cResettable
{
public:
	cLuaWorldTask(cPluginLua & a_Plugin, int a_FnRef) :
		cPluginLua::cResettable(a_Plugin),
		m_FnRef(a_FnRef)
	{
	}

	void operator()(cWorld & a_World)
	{
		cCSLock Lock(m_CSPlugin);
		if (m_Plugin != nullptr)
		{
			m_Plugin->Call(m_FnRef, &a_World);
		}
	}

protected:
	int m_FnRef;
};

Method1
{
	auto ResettableTask = std::make_shared<cLuaWorldTask>(*Plugin, FnRef);
	std::function<void(cWorld &)> CallableTask(std::placeholders::_1);
	CallableTask = ResettableTask;

	Plugin->AddResettable(ResettableTask);
	self->QueueTask(CallableTask);
}

Method2
{
	auto ResettableTask = std::make_shared<cLuaWorldTask>(*Plugin, FnRef);
	std::function<void(cWorld &)> CallableTask = ResettableTask;

	Plugin->AddResettable(ResettableTask);
	self->QueueTask(CallableTask);
}
```

`Severity	Code	Description	Project	File	Line
Error	C2893	Failed to specialize function template 'unknown-type std::invoke(_Callable &&,_Types &&...)' (compiling source file C:\Users\Tiger\Documents\GitHub\cuberite\src\Bindings\ManualBindings_World.cpp)	MCServer	C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\include\type_traits	1494
`